docs/persistence/influxdb.md

```diff
@@ -1,6 +1,6 @@
 # InfluxDB
 
-!!! info "Aiven discontinues InfluxDB"
+!!! warning "Deprecated"
     Aiven has informed us that they will discontinue support for InfluxDB.
     The timeline is yet to be confirmed, but could be as early as end-of-year 2022.
```

Comment:
Det er åpenbart ikke "end-of-year 2022" lenger. De har sagt noe om at når de setter en sluttdato vil den offentliggjøres minst 6 måneder før, så mulig vi bare skal si "The timeline is yet to be confirmed" og ikke sette noe dato på det.